components/pipeline-service/base/rbac/kustomization.yaml

```diff
@@ -4,5 +4,4 @@ resources:
   - cluster-role
   - openshift-pipelines
   # - pipelines-as-code
-  # - tekton-chains
```

Comment:
oh wait ... since this was commented out, we already have updated production in some sense ?

Reply:
That's what my understanding is as well. 
FYI @manish-jangra. Please let me know if that's no the case.